### PR TITLE
Translated README.md File To Hindi Language

### PR DESCRIPTION
**Pull request by everydaycodings**
Created a File named README-HI.md for the Hindi Translation of README.md File

### CODE REVIEW

**Comment by AryanMangale** [comment]
It seems like the translations are not very human, more like generic, formal language translations from Google.

**Comment by everydaycodings** [comment]
This is a Human Translation  not from google translation I have made this with the help of my Hindi Teacher.
Anyway you can check it.

File: README-HI.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+- [CSS Tricks An Open Source Etiquette Guidebook](https://css-tricks.com/open-source-etiquette-guidebook/) - केंट सी। डोड्स एंड सारा ड्रैसनर द्वारा लिखित एक ओपन सोर्स शिष्टाचार गाइडबुक।
+- [A to Z Resources for Students](https://github.com/dipakkr/A-to-Z-Resources-for-Students) - एक नई कोडिंग भाषा सीखने के लिए कॉलेज के छात्रों के लिए संसाधनों और अवसरों की घुमावदार सूची।
+- [Pull Request Roulette](http://www.pullrequestroulette.com/) - इस साइट में जीथब पर होस्ट किए गए ओपन सोर्स प्रोजेक्ट से संबंधित समीक्षा के लिए प्रस्तुत पुल अनुरोधों की एक सूची है।
+- ["How to Contribute to an Open Source Project on GitHub" by Egghead.io](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) - गिटहब पर ओपन सोर्स परियोजनाओं में योगदान कैसे शुरू करें, इसका चरण-दर-चरण वीडियो गाइड।
+- [Contributing to Open Source: A Live Walkthrough from Beginning to End](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - एक ओपन सोर्स योगदान के इस पूर्वाभ्यास में एक उपयुक्त प्रोजेक्ट को चुनने से लेकर एक मुद्दे पर काम करना, पीआर में विलय होने तक सब कुछ शामिल है।
```
Comment (vivek80801) [inline README-HI.md:63]: पीआर or पुल अनुरोध

File: README-HI.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+- ["How to get started with Open Source by" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - यह लेख अपनी पसंदीदा भाषा रुचि के आधार पर शुरुआती के लिए स्रोत खोलने में योगदान करने के लिए संसाधनों को शामिल करता है।
+- ["Browse good first issues to start contributing to open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - GitHub अब आपको ओपन सोर्स में योगदान देना शुरू करने के लिए पहले अच्छे मुद्दे खोजने में मदद करता है।
+- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - यह व्यापक लेख व्यवसायों की ओर निर्देशित है (लेकिन व्यक्तिगत योगदानकर्ताओं के लिए अभी भी उपयोगी है) जहां यह बात करता है कि क्यों, कैसे, और क्या-क्या ओपन-सोर्स प्रोजेक्ट्स में योगदान करना है।
+- ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) - ओपनसैट खेल की दुनिया में शुरू होता है, ओपनसोर्स खेल के मैदान में शुरू होता है। विशेष रूप से शिक्षा और व्यावहारिक अनुभव उद्देश्यों के लिए डिज़ाइन किया गया।
```
Comment (vivek80801) [inline README-HI.md:68]: ओपनसैट or ओपनसोर्स

File: README-HI.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+## डायरेक्ट गिटहब खोजता है
+Search links that point directly to suitable issues to contribute to on GitHub.
```
Comment (vivek80801) [inline README-HI.md:71]: ऐसे लिंक खोजें जो सीधे GitHub पर योगदान करने के लिए उपयुक्त मुद्दों की ओर इशारा करते हैं।

File: README-HI.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+- [is:issue is:open label:beginner](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Abeginner)
+- [is:issue is:open label:easy](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aeasy)
+- [is:issue is:open label:first-timers-only](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Afirst-timers-only)
+- [is:issue is:open label:good-first-bug](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Agood-first-bug)
+- [is:issue is:open label:"good first issue"](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A"good+first+issue")
+- [is:issue is:open label:starter](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astarter)
+- [is:issue is:open label:up-for-grabs](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
+
+## मोज़िला का योगदानकर्ता पारिस्थितिकी तंत्र
```
Comment (vivek80801) [inline README-HI.md:80]: पारिस्थितिकी तंत्र or इकोसिस्टम

File: README-HI.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+- [Good First Bugs](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - बग जो डेवलपर्स ने परियोजना के लिए एक अच्छा परिचय के रूप में पहचाना है।
```
Comment (vivek80801) [inline README-HI.md:81]: बग or मुद्दा

File: README-HI.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - बग जो एक संरक्षक को सौंपा गया है, जो आईआरसी पर आपकी मदद करेगा जब आप एक फिक्स पर काम करते समय फंस जाते हैं।
```
Comment (vivek80801) [inline README-HI.md:82]: बग or मुद्दा

File: README-HI.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) - Bugzilla पर बग ढूंढने के लिए समर्पित एक साइट।
+- [Firefox DevTools](http://firefox-dev.tools/) - फ़ायरफ़ॉक्स ब्राउज़र में डेवलपर टूल के लिए दायर की गई बग्स को समर्पित एक साइट।
+- [What Can I Do For Mozilla](https://whatcanidoformozilla.org/) - अपने कौशल सेट और रुचियों के बारे में सवालों के एक समूह का उत्तर देकर पता लगा सकते हैं कि आप क्या काम कर सकते हैं।
+- [Start Mozilla](https://twitter.com/StartMozilla) - एक ट्विटर अकाउंट जो मुद्दों के बारे में ट्वीट करता है जो मोज़िला इकोसिस्टम के लिए योगदानकर्ताओं के लिए उपयुक्त है।
+## नए ओपन सोर्स योगदानकर्ताओं के लिए उपयोगी लेख
+- [अपनी पहली ओपन सोर्स परियोजना को कैसे चुनें (और योगदान करें)](https://github.com/collections/choosing-projects) by [@GitHub](https://github.com/github)
+- [कैसे ठीक करने के लिए अपना पहला ओपन सोर्स बग ढूंढें](https://www.freecodecamp.org/news/finding-your-first-open-source-project-or-bug-to-work-on-1712f651e5ba/) by [@Shubheksha](https://github.com/Shubheksha)
+- [केवल पहले टाइमर](https://kentcdodds.com/blog/first-timers-only/) by [@kentcdodds](https://github.com/kentcdodds)
```
Comment (vivek80801) [inline README-HI.md:90]: पहलीबार योगदानकर्ताओ के लिए

File: README-HI.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+- [ओपन सोर्स पर दया लाएं](https://web.archive.org/web/20201009150545/https://www.hanselman.com/blog/bring-kindness-back-to-open-source) by [@shanselman](https://github.com/shanselman)
```
Comment (vivek80801) [inline README-HI.md:91]: ओपन सोर्स पर  दयालुता लाएं

File: README-HI.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+- [कैसे अपने कोड में एक बग को खोजने के लिए](https://8thlight.com/blog/doug-bradbury/2016/06/29/how-to-find-bug-in-your-code.html) by [@dougbradbury](https://twitter.com/dougbradbury)
+- [मास्टरींग मार्कडाउन](https://guides.github.com/features/mastering-markdown/) by [@GitHub](https://github.com/github)
+- [पहला मिशन: योगदानकर्ता पृष्ठ](https://medium.com/@forCrowd/first-mission-contributors-page-df24e6e70705#.2v2g0no29) by [@forCrowd](https://github.com/forCrowd)
+- [सिर्फ 5 मिनट में अपना पहला ओपन सोर्स योगदान कैसे करें](https://www.freecodecamp.org/news/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a/) by [@roshanjossey](https://github.com/Roshanjossey/)
+- [Hacktoberfest 2019: आप अपनी मुफ्त शर्ट कैसे प्राप्त कर सकते हैं - भले ही आप कोडिंग में नए हों](https://www.freecodecamp.org/news/hacktoberfest-2018-how-you-can-get-your-free-shirt-even-if-youre-new-to-coding-96080dd0b01b/) by [@quincylarson](https://www.freecodecamp.org/news/author/quincylarson/)
+- [एक बिटर गाइड टू ओपन सोर्स](https://medium.com/codezillas/a-bitter-guide-to-open-source-a8e3b6a3c1c4) by [@ken_wheeler](https://medium.com/@ken_wheeler)
```
Comment (vivek80801) [inline README-HI.md:99]: बिटर or बेहतर

File: README-HI.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+- [Pro Git](https://git-scm.com/book/en/v2) - संपूर्ण प्रो गिट पुस्तक, स्कॉट चाकोन और बेन स्ट्राब द्वारा लिखित और एप्रेस द्वारा प्रकाशित।
+- [Git-it](https://github.com/jlord/git-it-electron) - स्टेप बाय स्टेप गिट ट्यूटोरियल डेस्कटॉप ऐप।
+- [Flight Rules for Git](https://github.com/k88hudson/git-flight-rules) -जब चीजें गलत हो जाती हैं तो क्या करना है इसके बारे में एक गाइड।
+- [Git Guide for Beginners in Spanish](https://platzi.github.io/git-slides/#/) - Git और GitHub के बारे में स्लाइड्स का एक पूरा गाइड स्पेनिश में समझाया गया है। ऊना गुइया कम्पला दे दीपोसाइटिवस सोब्रे गिट वाई गिथहब एक्सपिकाडास एन Español।
+- [Git Kraken](https://www.gitkraken.com/git-client) - संस्करण नियंत्रण के लिए विज़ुअल, क्रॉस-प्लेटफ़ॉर्म, और इंटरैक्टिव `गिट` डेस्कटॉप एप्लिकेशन।
+- [Git Tips](https://github.com/git-tips/tips) - सबसे अधिक इस्तेमाल किए जाने वाले गिट टिप्स और ट्रिक्स का संग्रह।
+- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - कमिट अदर, परफेक्ट बाद, पब्लिश वन्स: बेस्ट प्रैक्टिस।
+- [Git Interactive Tutorial](https://learngitbranching.js.org/) - सबसे दृश्य और इंटरैक्टिव तरीके से Git सीखें।
+
+## स्रोत पुस्तकें खोलें
```
Comment (vivek80801) [inline README-HI.md:125]: ओपन सोर्स किताबे

File: README-HI.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+- [Open Sources 2.0 : The Continuing Evolution](https://archive.org/details/opensources2.000diborich) (2005) - ओपन सोर्स २.० आज के प्रौद्योगिकी नेताओं के व्यावहारिक और विचारोत्तेजक निबंधों का एक संग्रह है, जो १ ९९९ की पुस्तक, ओपन सोर्स: द रिव्यूज़ इन द रिवोल्यूशनरी में विकसित हुई विकासवादी तस्वीर को जारी रखता है।
+- [The Architecture of Open Source Applications](http://www.aosabook.org/en/git.html) - वितरित वर्कफ़्लोज़ को सक्षम करने के लिए कवर के तहत Git के विभिन्न पहलू काम करते हैं, और यह कैसे अन्य संस्करण नियंत्रण प्रणालियों (VCS) से भिन्न होता है।
+- [Open Sources: Voices from the Open Source Revolution](https://www.oreilly.com/openbook/opensources/book/) - लिनस टॉर्वाल्ड्स (लिनक्स), लैरी वॉल (पर्ल), और रिचर्ड स्टेलमैन (जीएनयू) जैसे ओपन-सोर्स अग्रदूतों से निबंध।
+
+## ओपन सोर्स योगदान पहल
+- [Up For Grabs](https://up-for-grabs.net/) - शुरुआत के अनुकूल मुद्दों के साथ परियोजनाएं शामिल हैं
+- [First Timers Only](https://www.firsttimersonly.com/) - कीड़े की एक सूची जिसे "पहले-टाइमर-केवल" लेबल किया जाता है।
```
Comment (vivek80801) [inline README-HI.md:135]: कीड़े?

**Review by vivek80801 — CHANGES_REQUESTED** [review]

**Comment by vivek80801** [comment]
"bugs" are not translated correctly. 
you can use मुद्दा insted of बग्स, कीड़े etc.

**Comment by erictleung** [comment]
@vivek80801 thanks for your comments and feedback!

If you would like to get GitHub commit credit for helping so much with these translations, might I suggest to use the [suggest changes](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) feature in GitHub? This will allow you to directly make those suggested change you have commented and @everydaycodings can more quickly incorporate those changes.

**Comment by everydaycodings** [comment]
> @vivek80801 thanks for your comments and feedback!
> 
> If you would like to get GitHub commit credit for helping so much with these translations, might I suggest to use the [suggest changes](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) feature in GitHub? This will allow you to directly make those suggested change you have commented and @everydaycodings can more quickly incorporate those changes.

Ya @vivek80801 use  [suggest changes](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change)  so that I can more quickly Incorporate those changes.

**Comment by vivek80801** [comment]
@erictleung  and @everydaycodings 

I don't know how to use that feature. I read docs on `Incorporating feedback in your pull request`. But I will need write access.

**Comment by everydaycodings** [comment]
Ok @vivek80801 when you Complete your editing then please notify me I will merge it to my Pull Request 

Thanku

**Comment by vivek80801** [comment]
@everydaycodings 
> Ok @vivek80801 when you Complete your editing then please notify me I will merge it to my Pull Request
> 
> Thanku

Can You explain what to do from my end ?

**Comment by erictleung** [comment]
@vivek80801 see below to access the review button which you have done.



But then you can access a suggestion button shown as the left-most symbols in the comment box. It has a plus-minus sign on a document.



After clicking on it, you should see a code block insertion that you can edit.



Source: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

File: README-HI.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+यह उन लोगों के लिए संसाधनों की एक सूची है जो ओपन सोर्स में योगदान करने के लिए नए हैं।
+
+यदि आपको अतिरिक्त संसाधन मिलते हैं, तो कृपया एक पुल अनुरोध में योगदान करें।
+
+यदि आपके कोई प्रश्न या टिप्पणी है, तो कृपया एक मुद्दा बनाएं।
+
+**अंतर्वस्तु**
+
+- [सामान्य रूप से ओपन सोर्स में योगदान करना](#contributing-to-open-source-in-general)
+- [डायरेक्ट गिटहब खोजता है](#direct-github-searches)
```
[the remark this answers is not in the document]
Comment (anmolsahu2k) [reply README-HI.md:40]: > I think this should be [डायरेक्ट गिटहब खोजें ](#direct-github-searches)
> 
> ```diff
> - [डायरेक्ट गिटहब खोजता है](#direct-github-searches)
> 
> + [डायरेक्ट गिटहब खोजें ](#direct-github-searches)
> ```

Yes this is more accurate

File: README-HI.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+- [मोज़िला का योगदानकर्ता पारिस्थितिकी तंत्र](#mozillas-contributor-ecosystem)
+- [नए ओपन सोर्स योगदानकर्ताओं के लिए उपयोगी लेख](#useful-articles-for-new-open-source-contributors)
+- [संस्करण नियंत्रण का उपयोग करना](#using-version-control)
+- [स्रोत पुस्तकें खोलें](#open-source-books)
```
[the remark this answers is not in the document]
Comment (anmolsahu2k) [reply README-HI.md:44]: Yes, this is more accurate

**Review by anmolsahu2k — CHANGES_REQUESTED** [review]
The comments have to be reviewed for changes.

File: README-HI.md [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+<!-- Do not translate this -->
+
+# आपका स्वागत है नौसिखिया ओपन सोर्स योगदानकर्ताओं!
+
+[![अनुरोध का स्वागत करें](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![पहली बार केवल अनुकूल](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![स्थिति बनाएँ](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+
+यह उन लोगों के लिए संसाधनों की एक सूची है जो ओपन सोर्स में योगदान करने के लिए नए हैं।
```
Comment (dumbcoder7) [inline README-HI.md:31]:
```suggestion
यह उन लोगों के लिए संसाधनों की एक सूची हैं जो ओपन सोर्स में योगदान करने के लिए नए हैं।
```

File: README-HI.md [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+
+यदि आपको अतिरिक्त संसाधन मिलते हैं, तो कृपया एक पुल अनुरोध में योगदान करें।
+
+यदि आपके कोई प्रश्न या टिप्पणी है, तो कृपया एक मुद्दा बनाएं।
```
Comment (dumbcoder7) [inline README-HI.md:35]:
```suggestion
यदि आपके कोई प्रश्न या टिप्पणी हैं, तो कृपया एक मुद्दा बनाएं।
```

File: README-HI.md [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+- [संस्करण नियंत्रण का उपयोग करना](#using-version-control)
+- [स्रोत पुस्तकें खोलें](#open-source-books)
+- [ओपन सोर्स योगदान पहल](#open-source-contribution-initiatives)
+- [लाइसेंस](#license)
+
+## सामान्य रूप से ओपन सोर्स में योगदान करना
+- [ओपन सोर्स में योगदान करने के लिए निश्चित गाइड](https://www.freecodecamp.org/news/the-definitive-guide-to-contributing-to-open-source-900d5f9f2282/) by [@DoomHammerNG](https://twitter.com/DoomHammerNG)
+- [ओपन सोर्स के लिए एक परिचय](https://www.digitalocean.com/community/tutorial_series/an-introduction-to-open-source) - GitHub पर सफलता में योगदान करने के लिए अपने रास्ते पर मार्गदर्शन करने के लिए DigitalOcean द्वारा ट्यूटोरियल।
+- [Issuehub.io](http://issuehub.io/) - लेबल और भाषा द्वारा GitHub मुद्दों को खोजने के लिए एक उपकरण।
+- [Code Triage](https://www.codetriage.com/) - अन्य, वास्तव में अच्छा, लोकप्रिय रिपॉजिटरी और भाषा द्वारा फ़िल्टर किए गए मुद्दों को खोजने के लिए उपकरण।
```
Comment (dumbcoder7) [inline README-HI.md:52]:
```suggestion
- [Code Triage](https://www.codetriage.com/) - लोकप्रिय रिपॉजिटरी और भाषा द्वारा फ़िल्टर किए गए मुद्दों को खोजने के लिए उपकरण।
```

File: README-HI.md [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+- [Awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners) - एक GitHub रेपो जो नए योगदानकर्ताओं के लिए अच्छे बग्स के साथ प्रोजेक्ट करता है, और उनका वर्णन करने के लिए लेबल लागू करता है।
+- [Open Source Guides](https://opensource.guide/) - उन व्यक्तियों, समुदायों और कंपनियों के लिए संसाधनों का संग्रह, जो ओपन सोर्स प्रोजेक्ट को चलाना और उसमें योगदान देना सीखना चाहते हैं।
+- [45 Github Issues Dos and Don’ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) -GitHub पर Do's और Don'ts।
```
Comment (dumbcoder7) [inline README-HI.md:55]:
```suggestion
- [45 Github Issues Dos and Don’ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) -GitHub के Do's और Don'ts।
```

File: README-HI.md [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+- [GitHub Guides](https://guides.github.com/) - कैसे प्रभावी रूप से GitHub का उपयोग करने के बारे में बुनियादी गाइड।
```
Comment (dumbcoder7) [inline README-HI.md:56]: Feels unnatural

```suggestion
- [GitHub Guides](https://guides.github.com/) - प्रभावी रूप से GitHub का उपयोग करने के बारे में बुनियादी गाइड।
```

File: README-HI.md [hunk 19]
```diff
@@ -0,0 +1,141 @@
...
+- [Contribute to Open Source](https://github.com/danthareja/contribute-to-open-source) - एक सिमुलेशन परियोजना के लिए कोड का योगदान करके GitHub वर्कफ़्लो जानें।
```
Comment (dumbcoder7) [inline README-HI.md:57]: - Corrected transliteration of 'Simulation'

- Better not to translate the word 'project'

```suggestion
- [Contribute to Open Source](https://github.com/danthareja/contribute-to-open-source) - एक सिम्युलेशन प्रोजेक्ट के लिए कोड का योगदान करके GitHub वर्कफ़्लो जानें।
```

File: README-HI.md [hunk 20]
```diff
@@ -0,0 +1,141 @@
...
+- [Linux Foundation's Open Source Guides for the Enterprise](https://www.linuxfoundation.org/resources/open-source-guides/) - लिनक्स फाउंडेशन के स्रोत स्रोत खोलने के लिए गाइड।
```
Comment (dumbcoder7) [inline README-HI.md:58]: Poorly translated

```suggestion
- [Linux Foundation's Open Source Guides for the Enterprise](https://www.linuxfoundation.org/resources/open-source-guides/) - लिनक्स फाउंडेशन कि तरफ से ओपन सोर्स प्रोजेक्ट्स के लिए गाइड।
```

File: README-HI.md [hunk 21]
```diff
@@ -0,0 +1,141 @@
...
+- [CSS Tricks An Open Source Etiquette Guidebook](https://css-tricks.com/open-source-etiquette-guidebook/) - केंट सी। डोड्स एंड सारा ड्रैसनर द्वारा लिखित एक ओपन सोर्स शिष्टाचार गाइडबुक।
```
Comment (dumbcoder7) [inline README-HI.md:59]: Fixed name transliteration

```suggestion
- [CSS Tricks An Open Source Etiquette Guidebook](https://css-tricks.com/open-source-etiquette-guidebook/) - केंट सी. डोड्स एवं सारा ड्रैसनर द्वारा लिखित एक ओपन सोर्स शिष्टाचार गाइडबुक।
```

File: README-HI.md (hunk 1 above)
Comment (dumbcoder7) [reply README-HI.md:63]: पुल रिक्वेस्ट Feels best

File: README-HI.md [hunk 22]
```diff
@@ -0,0 +1,141 @@
...
+- [A to Z Resources for Students](https://github.com/dipakkr/A-to-Z-Resources-for-Students) - एक नई कोडिंग भाषा सीखने के लिए कॉलेज के छात्रों के लिए संसाधनों और अवसरों की घुमावदार सूची।
+- [Pull Request Roulette](http://www.pullrequestroulette.com/) - इस साइट में जीथब पर होस्ट किए गए ओपन सोर्स प्रोजेक्ट से संबंधित समीक्षा के लिए प्रस्तुत पुल अनुरोधों की एक सूची है।
```
Comment (dumbcoder7) [inline README-HI.md:61]: Don't transliterate 'GitHub'

```suggestion
- [Pull Request Roulette](http://www.pullrequestroulette.com/) - इस साइट में GitHub पर होस्ट किए गए ओपन सोर्स प्रोजेक्ट से संबंधित समीक्षा के लिए प्रस्तुत पुल अनुरोधों की एक सूची है।
```

File: README-HI.md [hunk 23]
```diff
@@ -0,0 +1,141 @@
...
+- ["How to Contribute to an Open Source Project on GitHub" by Egghead.io](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) - गिटहब पर ओपन सोर्स परियोजनाओं में योगदान कैसे शुरू करें, इसका चरण-दर-चरण वीडियो गाइड।
+- [Contributing to Open Source: A Live Walkthrough from Beginning to End](https://medium.com/@kevinjin/contributing-to-open-source-walkthrough-part-0-b3dc43e6b720) - एक ओपन सोर्स योगदान के इस पूर्वाभ्यास में एक उपयुक्त प्रोजेक्ट को चुनने से लेकर एक मुद्दे पर काम करना, पीआर में विलय होने तक सब कुछ शामिल है।
+- ["How to Contribute to Open Source Project by" Sarah Drasner](https://css-tricks.com/how-to-contribute-to-an-open-source-project/) - वे गिटहब पर किसी और की परियोजना के लिए एक पुल अनुरोध (पीआर) का योगदान करने की नॉटी-ग्रिट्टी पर ध्यान केंद्रित कर रहे हैं।
```
Comment (dumbcoder7) [inline README-HI.md:64]:
```suggestion
- ["How to Contribute to Open Source Project by" Sarah Drasner](https://css-tricks.com/how-to-contribute-to-an-open-source-project/) - वे गिटहब पर किसी और की परियोजना के लिए एक पुल अनुरोध (पीआर) का योगदान करने पर ध्यान केंद्रित कर रहे हैं।
```

File: README-HI.md [hunk 24]
```diff
@@ -0,0 +1,141 @@
...
+- ["How to get started with Open Source by" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - यह लेख अपनी पसंदीदा भाषा रुचि के आधार पर शुरुआती के लिए स्रोत खोलने में योगदान करने के लिए संसाधनों को शामिल करता है।
```
Comment (dumbcoder7) [inline README-HI.md:65]:
```suggestion
- ["How to get started with Open Source by" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - यह लेख अपनी पसंदीदा भाषा रुचि के आधार पर नौसिखियों के लिए ओपन सोर्स में योगदान करने के लिए संसाधनों को शामिल करता है।
```

File: README-HI.md [hunk 25]
```diff
@@ -0,0 +1,141 @@
...
+- ["Browse good first issues to start contributing to open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - GitHub अब आपको ओपन सोर्स में योगदान देना शुरू करने के लिए पहले अच्छे मुद्दे खोजने में मदद करता है।
+- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - यह व्यापक लेख व्यवसायों की ओर निर्देशित है (लेकिन व्यक्तिगत योगदानकर्ताओं के लिए अभी भी उपयोगी है) जहां यह बात करता है कि क्यों, कैसे, और क्या-क्या ओपन-सोर्स प्रोजेक्ट्स में योगदान करना है।
```
Comment (dumbcoder7) [inline README-HI.md:67]:
```suggestion
- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - यह व्यापक लेख व्यवसायों की ओर निर्देशित है (लेकिन व्यक्तिगत योगदानकर्ताओं के लिए भी उपयोगी है) जहां यह बात करता है कि क्यों, कैसे, और क्या-क्या ओपन-सोर्स प्रोजेक्ट्स में योगदान करना उचित है।
```

File: README-HI.md (hunk 2 above)
Comment (dumbcoder7) [inline README-HI.md:68]:
```suggestion
- ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) - ओपनसोर्स खेल की दुनिया ओपनसोर्स प्लेग्राउंड में शुरू होती है। विशेष रूप से शिक्षा और व्यावहारिक अनुभव उद्देश्यों के लिए रचा गया हैं।
```

File: README-HI.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (anmolsahu2k) [reply README-HI.md:40]: > How about
> 
> ```diff
> - [डायरेक्ट गिटहब खोजें ](#direct-github-searches)
> 
> + [डायरेक्ट गिटहब खोजना](#direct-github-searches)
> ```

No, the correct one should be-
```diff
- [डायरेक्ट गिटहब खोजता है](#direct-github-searches)
+ [डायरेक्ट गिटहब खोजें](#direct-github-searches)
```

**Review by AniketSawale — CHANGES_REQUESTED** [review]
इस काम के तहत लाइसेंस प्राप्त है।[ क्रिएटिव कॉमन्स एट्रिब्यूशन-शेयरअलाइल 4.0 इंटरनेशनल लाइसेंस.](https://creativecommons.org/licenses/by-sa/4.0/)

File: README-HI.md [hunk 26]
```diff
@@ -0,0 +1,141 @@
...
+- [First Timers Only](https://www.firsttimersonly.com/) - कीड़े की एक सूची जिसे "पहले-टाइमर-केवल" लेबल किया जाता है।
+- [First Contributions](https://firstcontributions.github.io/) - 5 मिनट में अपना पहला ओपन सोर्स योगदान दें। एक उपकरण और ट्यूटोरियल शुरुआती योगदान करने में मदद करने के लिए। [यहाँ] (https://github.com/firstcontributions/first-contributions) साइट के लिए GitHub स्रोत कोड है और स्वयं भंडार में योगदान करने का अवसर है।
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - ओपन सोर्स योगदान को प्रोत्साहित करने के लिए एक कार्यक्रम। अक्टूबर के महीने में कम से कम 4 पुल अनुरोध करके टी-शर्ट और स्टिकर जैसे उपहार अर्जित करें।
+- [24 Pull Requests](https://24pullrequests.com) -24 पुल अनुरोध दिसंबर महीने के दौरान ओपन सोर्स सहयोग को बढ़ावा देने के लिए एक परियोजना है।
+
+## लाइसेंस
+<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />इस काम के तहत लाइसेंस प्राप्त है<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">क्रिएटिव कॉमन्स एट्रिब्यूशन-शेयरअलाइल 4.0 इंटरनेशनल लाइसेंस</a>.
```
Comment (AniketSawale) [inline README-HI.md:141]: <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons License</a><br />इस काम के तहत लाइसेंस प्राप्त है। <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">क्रिएटिव कॉमन्स एट्रिब्यूशन-शेयरअलाइल 4.0 इंटरनेशनल लाइसेंस</a>.

**Review by AniketSawale — APPROVED** [review]

**Comment by Jeevan-Kiran-Lenka** [comment]
Hey! I have some options in my mind for this since we are translating the page using a google translator some words like व्यक्तिगत योगदानकर्ताओं, पूर्वाभ्यास, योगदानकर्ताओं are not so familiar around Hindi community also Hindi people nowadays don't use that.

So instead of using hard words as shown above we can simplify it into bits that the general public can understand as when I was reading the readme I have to do google searches for Hindi words to get the meaning of that words

Other than that great work

**Review by saranshusa — APPROVED** [review]

**Review by Umesh-01 — APPROVED** [review]

**Pull request merged**